app/src/main/res/navigation/mobile_navigation.xml

```diff
@@ -0,0 +1,175 @@
+<?xml version="1.0" encoding="utf-8"?>
+<navigation xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/mobile_navigation"
+    app:startDestination="@id/nav_today_weather">
+
+    <fragment
```

Comment:
This file looks strange. Was it auto generated? 
I tried to remove all these actions and navigation continued to work well. As far as I remember, we need actions only to do navigation in our code, like this:  `view.findNavController().navigate(action)`
Let's try to get rid of all actions.

Reply:
I added all those actions. Because I was going to use animation when switching fragments. I forgot to remove it. Fixed